PubNubUnity/Assets/PubNub/Models/Server/BuildRequests.cs

```diff
@@ -217,6 +231,9 @@ public static Uri BuildFetchRequest (string[] channels, long start, long end, ui
             if (end != -1) {
                 parameterBuilder.AppendFormat ("&end={0}", end.ToString ().ToLowerInvariant ());
             }
+            if (withMeta) {
```

Comment:
Should this be `includeMeta`?

Reply:
Oh, for internal it shouldn't matter I thought it might show up in an IDE since that's the name of the param in the method signature. 

Carry on.